mangaki/mangaki/utils/common.py

```diff
@@ -1,24 +1,37 @@
 from django.conf import settings
 from mangaki.utils.chrono import Chrono
+from sklearn.metrics import mean_squared_error
 import pickle
 import os.path
 
 
+VERBOSE = False
 PICKLE_DIR = os.path.join(settings.BASE_DIR, '../pickles')
 
 
 class RecommendationAlgorithm:
     def __init__(self):
-        self.chrono = Chrono(True)
+        self.verbose = VERBOSE
```

Comment:
Why storing as an instance variable, a module constant?
`VERBOSE` value will never change, except if you reload the module (through dev server due to file changes).